doc/list.rst

```diff
@@ -2410,6 +2410,25 @@ List of Available Rules
    Part of rule sets `@PhpCsFixer:risky <./ruleSets/PhpCsFixerRisky.rst>`_ `@Symfony:risky <./ruleSets/SymfonyRisky.rst>`_
 
    `Source PhpCsFixer\\Fixer\\PhpUnit\\PhpUnitConstructFixer <./../src/Fixer/PhpUnit/PhpUnitConstructFixer.php>`_
+-  `php_unit_data_provider_name <./rules/php_unit/php_unit_data_provider_name.rst>`_
+
+   Data provider names must match the name of the test. Only safe scenarios are covered.
```

Comment:
>Only safe scenarios are covered.

Sound like a conflict/contradiction with the rule being `risky`, the fixing isn't "safe".

Reply:
"relatively safe" 😅. There is description of why it's risky.

Reply:
>There is description of why it's risky.

so lets go with that and drop this "safe" copy, I appreciate the effort to reduce the risky-ness of the rule, but it is not "safe".